README.md

```diff
@@ -285,7 +285,7 @@ Configuration documentation can be found at the [Instaclustr Terraform Registry]
 | KAFKA_REST_PROXY         | 5.0.0                                 | KAFKA                                                                              |
 | KAFKA_SCHEMA_REGISTRY    | 5.0.0, 5.0.4                          | KAFKA                                                                              |
 | KARAPACE_SCHEMA_REGISTRY | 2.1.2                                 | KAFKA <br/> **Not compatible with:** <br/> KAFKA_REST_PROXY, KAFKA_SCHEMA_REGISTRY |
-| OPENSEARCH               | 1.2.4                                 |                                                                                    |
+| OPENSEARCH               | 1.3.2, 2.0.0                          |                                                                                    |
 | ELASTICSEARCH (For Legacy Support Only)           | 1.13.3                                |                                                                                    |
```

Comment:
could probably fix up this line to be in-line with the rest

Reply:
I'm not exactly sure what you mean by in-line here Mitch?